CONTRIBUTING.md

```diff
@@ -46,6 +48,15 @@ This guide will help you get started and outline some things you should know whe
 * See [here](./documentation/development/testing.md) for how to run the tests
 
 
+## Developing
+
+* update depependencies: `glide up`
+* add a new Go library:
+  * update dependencies in a separate PR
+  * `glide get <package name>`
+  * add the source code of the added library, which is located in [vendor](vendor), to Git
```

Comment:
Yes, tell them that they should check in the source code of the dependency. Some Go projects don't do that, but given GT's long-term aim, I think we should, and developers should know that.